inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -171,6 +168,9 @@ target_compile_definitions(${TARGET_NAME} PUBLIC -DMKLDNN_THR=${MKLDNN_THR})
 target_link_libraries(${TARGET_NAME} PRIVATE mkldnn inference_engine inference_engine_legacy
                                              inference_engine_transformations inference_engine_lp_transformations openvino::conditional_compilation)
 
+target_include_directories(${TARGET_NAME} PRIVATE
+        $<TARGET_PROPERTY:mkldnn,INCLUDE_DIRECTORIES>)
```

Comment:
does `mkldnn` has `INTERFACE_INCLUDE_DIRECTORIES`?

Reply:
Will refactor in separate PR